styx-common/src/main/java/com/spotify/styx/model/Schedule.java

```diff
@@ -39,7 +39,7 @@
   public abstract String id();
 
   @JsonProperty
-  public abstract Partitioning partitioning();
+  public abstract Partitioning scheduleSpec();
```

Comment:
Hmm, I still feel like just calling it `schedule` makes most sense. Try reading the name in the context of [the docs here](https://github.com/spotify/styx/blob/2a05a2d7479a470cc057d2d738a6a8a581be7660/README.md#workflow-schedule-configuration) (from the cron PR). It would result in a configuration that looks like:

```yaml
schedules:
- id: foo
  schedule: @hourly
...
```

compared to

```yaml
schedules:
- id: foo
  schedule_spec: @hourly
...
```